src/System.IO.FileSystem/tests/Directory/GetDirectories_str_str.cs

```diff
@@ -304,7 +304,7 @@ public static void runTest()
 
             strArr = Directory.GetDirectories(Path.Combine(".", dirName), "*BB*");
             iCountTestcases++;
-            if (strArr.Length != 2)
+            if (strArr.Length != (Interop.IsLinux ? 1 : 2)) // Linux is case-sensitive
```

Comment:
How do you feel about using Interop.IsLinux / IsWindows in these tests rather than asking the FileSystem if it's case sensitive (to prepare for an OS with opposite than normal case sensitivity)?